src/components/AuthStoreSetter.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash/fp';
+import { useEffect } from 'react';
+import { AuthContextProps, useAuth } from 'react-oidc-context';
+import { loadAuthToken, OidcUser, processUser } from 'src/libs/auth';
+import { useOnMount } from 'src/libs/react-utils';
+import { authStore } from 'src/libs/state';
+
+function AuthStoreSetter(): null {
+  const auth: AuthContextProps = useAuth();
```

Comment:
i kinda hate that something called AuthContextProps has METHODS and not just be a json-like object.  But it looks to be coming from 3rd party, so meh...

Reply:
I agree that it is pretty ugly. We could make a type alias (sort of similar to what I did with OidcUser), and have any module outside of auth use the alias instead. That way `auth.ts` is the only module that needs to directly use `oidc-client-ts`